README.md

```diff
@@ -61,180 +22,101 @@ After you're finished please remove all the comments and instructions!
+- **[first project doing SQL databases]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- N/A
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
+Install PostgreSQL, and create a new database with: CREATE DATABASE vet_clinic;
+
+After just copy the code on schema.sql, the code on data.sql and finally the code on queries.sql
 
-To get a local copy up and running, follow these steps.
+After that just execute your code and get the same result as me! Thank you for reading this explanation!
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+- Have installed PostgreSQL
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+  cd desire-folder
+  git clone https://github.com/SaveryIV/vet-clinic.git
 
 ### Install
 
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
+No installation is needed.
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
+No command existing until now.
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+No test have been created yet for this project.
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
+No deployment have made yet.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Agustin Soria**
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Agustin-Soria](https://github.com/SaveryIV)
+- LinkedIn: [Agustin Soria](https://www.linkedin.com/in/agust%C3%ADn-ricardo-soria-meza-979747228/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[N/A]**
+- [ ] **[N/A]**
+- [ ] **[N/A]**
```

Comment:
-  [ ] You have done well customizing your readme file, Looking at your `future feature` section, I'll suggest you remove the `N/A` and add an appropriate feature that you are going to implement later on in this project. EG: `Optimizing query time`, `Create database diagram`. :+1: :wave:  
- [ ] Also I'll suggest you remove these placeholder comments as they were added there to help guide you while you customize your readme. :airplane: :small_airplane: 

- - Screenshot reference:
![Selection_011](https://github.com/SaveryIV/vet-clinic/assets/107632780/591ec92d-a5ff-4369-98ed-ee14fd941b86)